plugins/modules/cloudwatch_metric_alarm_info.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/python
+# This file is part of Ansible
+# GNU General Public License v3.0+ (see COPYING or https://wwww.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: cloudwatch_metric_alarm_info
+version_added: 5.0.0
+short_description: Gather information about the alarms for the specified metric
+description:
+  - Retrieves the alarms for the specified metric.
+author:
+  - Mandar Vijay Kulkarni (@mandar242)
+options:
+  alarm_names:
+    description:
+      - The name of the metric.
+    required: false
+    type: list
+    elements: str
+
+extends_documentation_fragment:
+- amazon.aws.aws
+- amazon.aws.ec2
+'''
+
+EXAMPLES = '''
+# Note: These examples do not set authentication details, see the AWS Guide for details.
+
+- name: describe the metric alarm based on metric name and namespace
+  amazon.aws.cloudwatch_metric_alarm_info:
+    cloudwatch_metric_alarm_info:
+        alarm_names:
+            - my-test-alarm-1
+            - my-test-alarm-2
+
+'''
+
+RETURN = '''
+metric_alarms:
+    description: The gathered information about specified metric alarms.
+    returned: when success
+    type: list
+    elements: dict
```

Comment:
Documented response seems incomplete to me https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudwatch.html#CloudWatch.Client.describe_alarms

Reply:
Added.